pkgs/development/libraries/qt-5/modules/qtbase.nix

```diff
@@ -39,6 +40,7 @@ stdenv.mkDerivation {
 
   name = "qtbase-${version}";
   inherit qtCompatVersion src version;
+  inherit debug;
```

Comment:
What does this option do? In https://github.com/NixOS/nixpkgs/pull/109239/files#diff-a178028d8bf3d9cb5c68fe3e5856be45f904bc22e1b9c6f938ed3c7c386026fdR259 there was a configure flag require to have debug symbols.

Reply:
I don't know, that didn't used to be the case. This isn't a new option. The only thing "new" here, is that Qt will remember which option it was built with so that we don't have to pass a debug flag into every package that depends on it.